internal/osbuildconfig/osbuildconfig_handler.go

```diff
@@ -0,0 +1,78 @@
+package osbuildconfig
+
+import (
+	"github.com/project-flotta/osbuild-operator/internal/manifests"
+	repositoryosbuild "github.com/project-flotta/osbuild-operator/internal/repository/osbuild"
+	repositoryosbuildconfig "github.com/project-flotta/osbuild-operator/internal/repository/osbuildconfig"
+	"github.com/project-flotta/osbuild-operator/internal/repository/secret"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+type OSBuildConfigHandler struct {
+	OSBuildConfigRepository repositoryosbuildconfig.Repository
+	SecretRepository        secret.Repository
+	OSBuildRepository       repositoryosbuild.Repository
+	Scheme                  *runtime.Scheme
+	OSBuildCRCreator        manifests.OSBuildCRCreatorInterface
+}
+
+func NewOSBuildConfigHandler(osBuildConfigRepository repositoryosbuildconfig.Repository, osBuildRepository repositoryosbuild.Repository, secretRepository secret.Repository, scheme *runtime.Scheme, osBuildCRCreator manifests.OSBuildCRCreatorInterface) *OSBuildConfigHandler {
+	return &OSBuildConfigHandler{
+		OSBuildConfigRepository: osBuildConfigRepository,
+		OSBuildRepository:       osBuildRepository,
+		SecretRepository:        secretRepository,
+		Scheme:                  scheme,
+		OSBuildCRCreator:        osBuildCRCreator,
+	}
+}
```

Comment:
Did you run `make fmt`? There should be an empty line here

Reply:
yes, it didn't change something

Reply:
Strange. Anyhow, there should be an empty line here